test/integration/admin_test.js

```diff
@@ -24,7 +24,8 @@ var RingpopClient = require('../../client.js');
 var testRingpopCluster = require('../lib/test-ringpop-cluster.js');
 
 testRingpopCluster({
-    size: 1
+    size: 1,
+    dontWaitForConvergence: true,
```

Comment:
I found this a little bit more understandable. This will never be set to false since the default is to wait for convergence. But it's no problem to change.